rdkit/Chem/Fraggle/FraggleSim.py

```diff
@@ -135,11 +132,11 @@ def select_fragments(fragments, ftype, hac):
       # needs to be greater 3 heavy atoms and greater than 40% of parent mol
       if isValidRingCut(fMol):
         result_hcount = fMol.GetNumAtoms()
-        if (result_hcount > 3) and (result_hcount > 0.4 * hac):
-          result = f
+        if result_hcount > 3 and result_hcount > 0.4 * hac:
+            result = f
```

Comment:
This looks like the indentation is now 4 spaces

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.